stock_move_location/i18n/stock_move_location.pot

```diff
@@ -0,0 +1,168 @@
+# Translation of Odoo Server.
```

Comment:
I think the PRs for new modules must not include a pot file these days as this is generated by weblate. 

@sbidoul @pedrobaeza do you confirm?

(BTW maybe we should have an automated test for this)

Reply:
.pot will be (re)generated by travis after merge. It does not harm if one is provided in the PR.

The only thing we try to avoid is PRs to **existing** modules that update **.po**, because that can generate merge conflicts with weblate.